modules/tinymce/src/plugins/lists/test/ts/browser/BackspaceDeleteInlineTest.ts

```diff
@@ -1,71 +1,60 @@
-  SilverTheme();
-
-  suite.test('TestCase-TBA: Lists: Backspace at beginning of LI on body UL', (editor) => {
-    editor.focus();
-    editor.selection.setCursorLocation(editor.getBody().firstChild.firstChild, 0);
-    editor.plugins.lists.backspaceDelete();
-    LegacyUnit.equal(DomQuery('#lists ul').length, 3);
-    LegacyUnit.equal(DomQuery('#lists li').length, 3);
-  });
-
-  suite.test('TestCase-TBA: Lists: Delete at end of LI on body UL', (editor) => {
-    editor.focus();
-    editor.selection.setCursorLocation(editor.getBody().firstChild.firstChild, 1);
-    editor.plugins.lists.backspaceDelete(true);
-    LegacyUnit.equal(DomQuery('#lists ul').length, 3);
-    LegacyUnit.equal(DomQuery('#lists li').length, 3);
-  });
+import Theme from 'tinymce/themes/silver/Theme';
 
-  const teardown = (editor, div) => {
-    editor.remove();
-    div.parentNode.removeChild(div);
-  };
-
-  const setup = (success, failure) => {
+describe('browser.tinymce.plugins.lists.BackspaceDeleteInlineTest', () => {
+  const setupElement = () => {
     const div = document.createElement('div');
-
     div.innerHTML = (
       '<div id="lists">' +
       '<ul><li>before</li></ul>' +
       '<ul id="inline"><li>x</li></ul>' +
       '<ul><li>after</li></ul>' +
       '</div>'
     );
-
     document.body.appendChild(div);
 
-    EditorManager.init({
-      selector: '#inline',
-      inline: true,
-      add_unload_trigger: false,
-      skin: false,
-      plugins: 'lists',
-      disable_nodechange: true,
-      init_instance_callback: (editor) => {
-        Pipeline.async({}, Log.steps('TBA', 'Lists: Backspace delete inline tests', suite.toSteps(editor)), () => {
-          teardown(editor, div);
-          success();
-        }, failure);
+    return {
+      element: SugarElement.fromDom(div),
+      teardown: () => {
+        document.body.removeChild(div);
       },
-      valid_styles: {
-        '*': 'color,font-size,font-family,background-color,font-weight,font-style,text-decoration,float,' +
-        'margin,margin-top,margin-right,margin-bottom,margin-left,display,position,top,left,list-style-type'
-      }
-    });
+    };
   };
 
-  setup(success, failure);
+  const hook = TinyHooks.bddSetupFromElement<Editor>({
+    selector: '#inline',
+    inline: true,
+    add_unload_trigger: false,
+    skin: false,
+    plugins: 'lists',
+    disable_nodechange: true,
+    valid_styles: {
+      '*': 'color,font-size,font-family,background-color,font-weight,font-style,text-decoration,float,' +
+      'margin,margin-top,margin-right,margin-bottom,margin-left,display,position,top,left,list-style-type'
+    }
+  }, setupElement, [ Plugin, Theme ], true);
+
+  it('TBA: Backspace at beginning of LI on body UL', () => {
+    const editor = hook.editor();
+    const body = editor.getBody();
+    editor.selection.setCursorLocation(body.firstChild.firstChild, 0);
+    editor.plugins.lists.backspaceDelete();
+    LegacyUnit.equal(DomQuery('#lists ul').length, 3);
```

Comment:
`assert.lengthOf` here and the places below instead please.

- [ ] Remove `LegacyUnit` test assertions